tests/Microsoft.Bot.Builder.TestBot.Json/Samples/ToDoBot/.vscode/launch.json

```diff
@@ -5,7 +5,7 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "type": "json",
+            "type": "bot",
```

Comment:
>bot [](start = 21, length = 3)

Is that an actual type?

Reply:
yes, it is. The type name is the same with the type name of adaptive-tool.
Originally, ".dialog" is the only debug target, so the type name is "json"(.dialog's default schema format) 
Now, we add LG into the debugging tool, So, type "json" should be dropped.